<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ATOM</class>
 <widget class="QMainWindow" name="ATOM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
background:rgba(30, 33, 36, 255);
border: 0px;
border-radius: 0px;
}
QWidget{
background: rgba(51, 57, 73, 255);
border-radius: 10px;
font-family: Poppins;
font-size: 16px;
font-weight: 400;
color: rgba(255, 255, 255, 255);
}
QToolButton{
border: 0px;
background: rgba(0, 128, 255, 255);
color: rgba(255, 255, 255, 255);
font-weight: 400;
border-radius: 4px;
}
QToolButton:pressed{
border: 2px solid transparent;
}
QToolButton#extractButton{
background: transparent;
padding-left:20px;
}
QToolButton#exitButton{
padding-left:35px;
}
QToolButton#pickFileButton{
padding-left:10px;
}
QPushButton{
border: 0px;
background: rgba(0, 128, 255, 255);
color: rgba(255, 255, 255, 255);
font-weight: 400;
border-radius: 4px;
}
QPushButton:pressed{
border: 2px solid transparent;
}
QPushButton#nextButton, #previousButton, #clearAllButton{
background: transparent;
}
QPushButton#cancelSelection{
background: rgba(255, 255, 255, 255);
color: rgba(0, 128, 255, 255);
}
QLabel#titleLabel,#pageLabel{
color: rgb(255, 255, 255);
background: transparent;
}
QListWidget::item{
height: 30px;
border: 1px solid white;
border-radius: 10px;
}
QListWidget::item:selected{
background: rgba(0, 128, 255, 64);
}
QListWidget{
padding: 20 5 10 5;
}s</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <property name="spacing">
     <number>10</number>
    </property>
    <item row="1" column="1">
     <widget class="QPushButton" name="previousButton">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>UI/resources/left.png</normaloff>UI/resources/left.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="7">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>80</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Blueprint Reader</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="exitButton">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string> Exit</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>UI/resources/exit.png</normaloff>UI/resources/exit.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pickFileButton">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Open File</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>UI/resources/open-file.png</normaloff>UI/resources/open-file.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="6">
     <widget class="QPushButton" name="nextButton">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>UI/resources/right.png</normaloff>UI/resources/right.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0" rowspan="2" colspan="7">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>80</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="pageLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Page 1/100</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="makeSelection">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Select Region</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="applySelection">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Apply Selection</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="4">
     <widget class="QWidget" name="fileDisplayWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>580</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3"/>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="sideWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignHCenter">
        <widget class="QToolButton" name="extractButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string> Extract</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>UI/resources/extract.png</normaloff>UI/resources/extract.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="selectionsList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::CopyAction</enum>
         </property>
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="uniformItemSizes">
          <bool>true</bool>
         </property>
         <property name="selectionRectVisible">
          <bool>false</bool>
         </property>
         <property name="itemAlignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="clearAllButton">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Clear All</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
